Rubberduck.Resources/Settings/UnitTestingPage.resx

```diff
@@ -129,6 +129,9 @@
   <data name="UnitTestSettings_BindingMode" xml:space="preserve">
     <value>Binding mode:</value>
   </data>
+  <data name="UnitTestSettings_DualBinding" xml:space="preserve">
```

Comment:
This might be beyond the scope of the PR but now that we're splitting up the resx files, the prefix seems redundant and probably should be gotten rid of at some point in future.

Reply:
Agreed.  The resx files are generally in bad need of clean-up and rearranging. That's a fairly monumental task in and of itself though. Maybe it needs an issue?

Reply:
[Left a comment](https://github.com/rubberduck-vba/Rubberduck/issues/4541#issuecomment-464053990) on #4541 .